wurst/file/Base64.wurst

```diff
@@ -0,0 +1,270 @@
+package Base64
+
+import Bitwise
+import StringUtils
+import ByteBuffer
+import ChunkedString
+import Execute
+import ErrorHandling
+
+/*  Base64 package provides an encoder/decoder pair that allows serializing and deserializing of any data.
+
+    Encode Data iteratively:
```

Comment:
Phrasing: I think `iteratively` makes this less clear since that's a bit of an overloaded term. I'd describe this as providing a builder pattern.